src/app/core/entity/default-field-value/handle-default-values.usecase.ts

```diff
@@ -20,41 +22,103 @@ export class HandleDefaultValuesUseCase {
     private logger: LoggingService,
   ) {}
 
-  handleFormGroup(
-    formGroup: FormGroup,
-    fieldConfigs: [string, EntitySchemaField][],
+  async handleExtendedEntityForm<T extends Entity>(
+    form: ExtendedEntityForm<T>,
+    entitySchema: EntitySchema,
     isNew: boolean,
-  ) {
-    fieldConfigs.forEach(([fieldName, fieldSchema]) => {
-      let defaultValueConfig = fieldSchema.defaultValue;
+  ): Promise<void> {
+    let inheritedConfigs: Map<string, DefaultValueConfig> =
+      this.getConfigsByMode(form.defaultValueConfigs, ["inherited"]);
+
+    const linkedEntityRefs: Map<string, string[]> = this.getLinkedEntityRefs(
+      inheritedConfigs,
+      form,
+    );
+
+    await this.loadLinkedEntitiesIntoForm(
+      Array.from(linkedEntityRefs.values()).flat(),
+      form,
+    );
+
+    this.enableChangeListener(form);
```

Comment:
do we need to enable change listeners for cases other than "inherited"?

Reply:
For loading the linked parent entities it would be fine to just listen to all properties from `defaultValues.map(dv -> localField)`

For updating the sync state, we need to listen to all `defaultValues.keys()`